compose/reference/run.md

```diff
@@ -26,24 +26,24 @@ Runs a one-time command against a service. For example, the following command st
 
     docker-compose run web bash
 
-Commands you use with `run` start in new containers with the same configuration as defined by the service' configuration. This means the container has the same volumes, links, as defined in the configuration file. There are two differences though.
+Commands you use with `run` start in new containers with the configuration defined by the service's configuration. This means that the resulting container has the same volumes, links, etc., as defined in the configuration file. However, there are two important differences.
```

Comment:
Avoid "etc". Maybe "...defined by the service's configuration, including volumes, links, and other configuration details. However..."